stories/molecules/Navigation.tsx

```diff
@@ -17,12 +17,12 @@ export const Navigation = (args) => (
             <NavItem {...args}>
                 <a href="/">Active item</a>
             </NavItem>
-            <NavItem useActiveClass pullRight>
-                <NavLink exact to="/">
+            <NavItem useActiveClass pullRight routerVersion={6}>
+                <NavLink end to="/somewhere">
                     Item on the right
                 </NavLink>
             </NavItem>
-            <NavItem useActiveClass>
+            <NavItem useActiveClass routerVersion={6}>
```

Comment:
The version is not selected in the "Controls" Storybook by default. When I choose 6 version I see item without styles. Is it ok?
![Screenshot 2022-10-27 at 13 02 13](https://user-images.githubusercontent.com/19436304/198268485-bc100eb5-2e0c-4b75-97a8-6ee61191e83e.png)